### DISCUSSION TITLE
An error occurred while reasoning using lightning inference.py,And the test did not save the visualization results after the training was completed

### DISCUSSION THREAD

**Discussion by luoyq6** [open-edge-platform/anomalib#1887] [Q&A] [answered]
### Describe the bug

 from . import _imaging as core ImportError: /lib64/libstdc++.so.6: version `CXXABI_1.3.9' not found (required by /opt/workspace/.conda/envs/anomal/lib/python3.10/site-packages/PIL/../../.././libLerc.so)

### Dataset

Folder

### Model

Other (please specify in the field below)

### Steps to reproduce the behavior

efficientad

### OS information

efficientad/

### Expected behavior

efficientad/

### Pip/GitHub

GitHub

### Configuration YAML

```yaml
efficientad/
```

### Logs

```shell
efficientad/
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by luoyq6** [comment]
@samet-akcay

**Comment by blaz-r** [comment]
Can you please share more of the error message

**Comment by luoyq6** [comment]
When I perform python tools/inference/lightning_inference py the following errors in order
![image](https://github.com/openvinotoolkit/anomalib/assets/110521846/7bdbbb35-9730-43af-ab99-2a09b678cc15)
@blaz-r

**Comment by blaz-r** [comment, marked as answer]
This looks like some of the packages in your environment are not correctly installed. I recommend that you make a fresh environment and follow [installation guide](https://github.com/openvinotoolkit/anomalib?tab=readme-ov-file#-installation) for installing from source to get the latest version of everything.
